src/GraphQL.ApiTests/GraphQL.NewtonsoftJson.approved.txt

```diff
@@ -1,9 +1,17 @@
 namespace GraphQL.NewtonsoftJson
 {
+    public class ExecutionErrorJsonConverter : Newtonsoft.Json.JsonConverter
+    {
+        public ExecutionErrorJsonConverter(GraphQL.Execution.IErrorInfoProvider errorInfoProvider) { }
+        public override bool CanRead { get; }
+        public override bool CanConvert(System.Type objectType) { }
+        public override object ReadJson(Newtonsoft.Json.JsonReader reader, System.Type objectType, object? existingValue, Newtonsoft.Json.JsonSerializer serializer) { }
+        public override void WriteJson(Newtonsoft.Json.JsonWriter writer, object? value, Newtonsoft.Json.JsonSerializer serializer) { }
+    }
     public class ExecutionResultJsonConverter : Newtonsoft.Json.JsonConverter
     {
-        public ExecutionResultJsonConverter(GraphQL.Execution.IErrorInfoProvider errorInfoProvider) { }
-        public ExecutionResultJsonConverter(GraphQL.Execution.IErrorInfoProvider errorInfoProvider, Newtonsoft.Json.Serialization.NamingStrategy? namingStrategy) { }
+        public ExecutionResultJsonConverter() { }
+        public ExecutionResultJsonConverter(Newtonsoft.Json.Serialization.NamingStrategy? namingStrategy) { }
```

Comment:
This decomposition is beneficial to the code - it became less parameters combinations.